tests/cxx/namespace_macro.cc

```diff
@@ -0,0 +1,32 @@
+//===--- namespace_macro.cc -----------------------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "tests/cxx/namespace_macro-direct.h"
+
+// IWYU_ARGS: -I . -DMYNS=myns
+
+int main() {
+  // IWYU: myns::foo is defined in "tests/cxx/namespace_macro-indirect.h", which isn't directly #included.
```

Comment:
We usually use an abbreviated form: `// IWYU: myns::foo is...*namespace-macro-indirect.h`.

See the rest of the test suite for examples.

Reply:
Done